aten/src/ATen/templates/Tensor.h

```diff
@@ -257,6 +267,7 @@ class CAFFE2_API Tensor {
 
   /// Returns a `Tensor`'s dimension names data structure
   NamedTensorMeta* get_named_tensor_meta() const;
+  NamedTensorMeta* get_named_tensor_meta();
```

Comment:
c++ question .... why don't these function signatures conflict?

Reply:
c++ lets you overload member methods based on const-ness.

Reply:
Err I'm changing the signatures to `const NamedTensorMeta* get_named_tensor_meta() const;`. I am not sure if this is actually different.